test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs

```diff
@@ -445,34 +423,25 @@ void ProcessExport(Batch<Metric> batch)
             }
 
             metricReader.Collect();
-            Assert.Equal(AggregatorStore.MaxMetricPoints, metricPointCount);
+            Assert.Equal(AggregatorStore.MaxMetricPoints, MetricPointCount());
 
-            metricPointCount = 0;
             metricReader.Collect();
-            Assert.Equal(AggregatorStore.MaxMetricPoints, metricPointCount);
+            Assert.Equal(AggregatorStore.MaxMetricPoints * 2, MetricPointCount());
 
             // These updates would be dropped.
+            // TODO review this
             counterLong.Add(10, new KeyValuePair<string, object>("key", "valueA"));
             counterLong.Add(10, new KeyValuePair<string, object>("key", "valueB"));
             counterLong.Add(10, new KeyValuePair<string, object>("key", "valueC"));
-            metricPointCount = 0;
             metricReader.Collect();
-            Assert.Equal(AggregatorStore.MaxMetricPoints, metricPointCount);
+            Assert.Equal(AggregatorStore.MaxMetricPoints * 3, MetricPointCount());
```

Comment:
Is using these multiples of 2 and 3 for the subsequent assertions fine, or should the metrics be flushed in some way?

Reply:
wont clearing metricItems help?